Prog/MAR_summary_units.sas

```diff
@@ -0,0 +1,146 @@
+/**************************************************************************
+ Program:  MAR_summary_units.sas
+ Library:  MAR
+ Project:  Urban-Greater DC
+ Author:   Rob and Yipeng
+ Created:  9/26/2018
+ Version:  SAS 9.4
+ Environment:  Local Windows session (desktop)
+ 
+ Description:  Create summary file of units from MAR address points. 
+
+ Modifications:
+**************************************************************************/
+
+%include "L:\SAS\Inc\StdLocal.sas";
+
+** Define libraries **;
+%DCData_lib( MAR )
+%DCData_lib( realprop )
+
+%let address_pt_date = 2018_06;
+%let revisions= New file;
+
+
+proc sort data = mar.address_points_&address_pt_date. out = addpt_in; by ssl; run;
+proc sort data = realprop.parcel_base out = parcel_in; by ssl; run;
+
+
+/* Set Address Points dataset from most recent update */
+data mar_units;
+	merge addpt_in (in=a)
+		  parcel_in (in=b where=(in_last_ownerpt=1));
```

Comment:
@rpitingolo - i added this restriction - can you chekc - i think it's included when we do the num-units file from ownerpoint.